plugins/generator-1.18.2/forge-1.18.2/procedures/entity_set_ticks_frozen.java.ftl

```diff
@@ -0,0 +1 @@
+${input$entity}.setTicksFrozen((int) ${input$ticks});
```

Comment:
Do not cast directly, but use `opt.toInt()` so it optimizes cast when needed and when not, same for 1.19.2